simple/test.js

```diff
@@ -3113,13 +3168,13 @@ exports.test = function (global) {
         };
 
         if (global.tiny) {
-          options.collections.push({ name: "values1000", label: "1k", size: 1000});
+          options.collections.push({ name: "valuesForSearch100000", label: "100k", size: 100000});
         } else if (global.small) {
-          options.collections.push({ name: "values10000", label: "10k", size: 10000});
+          options.collections.push({ name: "valuesForSearch1000000", label: "1000k", size: 1000000});
         } else if (global.medium) {
-          options.collections.push({ name: "values100000", label: "100k", size: 100000 });
+          options.collections.push({ name: "valuesForSearch10000000", label: "10000k", size: 10000000});
```

Comment:
```suggestion
          options.collections.push({ name: "valuesForSearch10000000", label: "10M", size: 10000000});
```
NIT: that is more readable